test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/ChainingSearchTests.cs

```diff
@@ -451,7 +455,11 @@ protected override async Task OnInitializedAsync()
                 {
                     Meta = new Meta { Tag = new List<Coding> { new Coding("testTag", Tag) } },
                     Type = Group.GroupType.Person,
+#if !R5
```

Comment:
What I don't like about the conditionals in this way is that its likely going to be this way from 5+, so this code will break on every new versions too

Reply:
tldr is we have a known number of previous versions and an unknown number of future versions :)